Translation corrections (#193)
07th-mod/higurashi-console-arcs: Update/tsuk_001.txt

//rつい言葉がきつくなったせいか、梨花ちゃんは怯えたように肩をすくませた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "つい言葉がきつくなったせいか、梨花ちゃんは怯えたように肩をすくませた。",
			NULL, "As my words grew more piercing, Rika-chan's shoulders tensed as though in fear.", Line_Normal);
	ClearMessage();

//rそれでも、俺と視線を合わせようともしない。ただ諦めたような表情で、……口を引き結んでいる。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それでも、俺と視線を合わせようともしない。ただ諦めたような表情で、……口を引き結んでいる。",

> 			NULL, "Still, she wouldn't look into my eyes. With a look like she was giving up... she closed her mouth.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Still, she wouldn't look me in the eyes. With a look like she was giving up... she closed her mouth.", Line_Normal);
```
